src/data/location/Radar.ts

```diff
@@ -63,10 +63,13 @@ class RadarLocation {
   private callback: Function | undefined;
   private backupUserId: string = 'cd66931c-a623-11ec-b909-0242ac120002';
 
-  async start() {
+  public async init() {
     const participantCode = await ParticipantCode.loadCurrentParticipantCodeFromLocalStorage();
     const userId: string = participantCode ?? this.backupUserId;
     this.configureRadar(userId);
+  }
+
+  public async start() {
     this.handleLocationPermission().then(async (granted) => {
       if (granted) {
         console.log('Location is granted!');
```

Comment:
Ik krijg bij het toestemming geven 3x deze melding en bij het opnieuw opstarten van de app altijd 2x deze melding. Zit denk ik iets dubbel in.